products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,274 @@
+and according to the rules. The Transfer window is a period during the year
+in which a football team can transfer players. There are usually two transfer windows
+in the same season, one in winter that lasts through January,
+and one in summer that lasts through July and August.
+
+Usually some sort of compensation is paid for the player's rights,
+which is known as a transfer fee. When a player moves from one team to another,
+their old contract is terminated and they negotiate a new one with the team
+they are moving to. In some cases, however, transfers can function similarly
+to player trades, as teams can offer another player on their team as part of the fee.
+
+Of course you may presume, there is a lot of money in the game.
+
+According to FIFA, from January to September 2018 there were 15,049 international transfers
+of male players with fees totaling US$ 7.1 billion dollars, and 577 international transfers
+of female players for US$ 493,235.
+
+Also, one last thing to consider is the season. 
+
+In an organized sports league, a typical season is the part of one year in which regulated games
+of the sport are in session. In football, it is generally from August or September to May,
+although in some countries - such as Northern Europe or East Asia - the season starts in the spring 
+and finishes in autumn, mainly due to weather conditions encountered during the winter.
+
+
+### Data Model
+
+In this article, we will present a graph model of a large-sized dataset
+of football transfers that include the most famous leagues. When we say large-sized dataset,
+what we think by that is the dataset that has all transfers from season 1992/1993 to season 2019/2020
+in nine leagues that are supported. Those leagues are Eredivisie, English Championship,
+English Premier League, French Ligue 1, German Bundesliga, Italian Serie A, Portuguese Liga Nos,
+Russian Premier Liga and Spanish Primera Division.
+
+
+* Each Football transfer has a team which makes a transfer and team to which player is transferred to,
+ so that will be our first node &mdash `Team`. Each team has a name so we will add the label `name`. 
+Also, most of the teams in our dataset will have the league in which they compete so we will add label `league`.
+* Following what we just described, there is also a player that is part of the transfer and that will be our
+next node &mdash `Player`. That nods will also have the label `name` and label `position`,
+but the label `age` will be added to the new node that we will create, and that is node `Transfer`.
+This node will be a connection between nodes `Player` and two nodes of type `Team` that take part
+in the transfer.  Node `Transfer` will also have a  label `fee`.
+Fee represents millions of euros that were paid as compensation for the player's rights.
+* The Last node in our model will be `Season`. It will have labels `season` and `year`.
+The label `season` will be in string format and label `year` in number format.
```

Comment:
```suggestion
The model consists of the following nodes:

* `Team` - a football team with a property `name` (e.g. `"FC Barcelona"`).
* `Player` - a professional football player, contains properties `name` (e.g. `"Luka Modric"`) and `position` (e.g. `"Central Midfield"`).
* `League` - a football league where multiple teams belong to, contains one property `name` (e.g. `"Premier League"`).
* `Transfer` - represents a football transfer that connects a `Player` that is transferred from one `Team` to another `Team` within a `Season`. Transfer contains one optional property `fee` (e.g. `195.50`) that represents a transfer fee in millions of euros.
* `Season` - a football season with two properties `name` (e.g. `"2019/2020"`) and `year` (e.g. `2019`)
```

Reply:
There were few issues in this paragraph. `labels` where used even though it was describing `properties`. Labels are node names, like `Team`, `Player`, .... Properties are node properties like `name`, `year`, ...

Reply:
:+1: